tensorflow/compiler/tf2tensorrt/utils/trt_engine_utils.cc

```diff
@@ -37,9 +37,24 @@ namespace tensorrt {
 using absl::StrCat;
 
 ExecutionContext ExecutionContext::Create(nvinfer1::ICudaEngine* cuda_engine) {
+  bool has_int32_output = false;
+  for (int i = 0; i < cuda_engine->getNbBindings(); i++) {
+    if (!cuda_engine->bindingIsInput(i) &&
+        cuda_engine->getBindingDataType(i) == nvinfer1::DataType::kINT32) {
+      has_int32_output = true;
+      break;
+    }
+  }
+  if (!IS_TRT_VERSION_GE(8, 0, 0, 0) && has_int32_output) {
+    // Workaround for TRT 7 bug with shape output (nvbugs/3390469).
```

Comment:
Let's turn this comment into a TODO:
// TODO(nvbugs/3390469): Remove this workaround when the bug is fixed.

Reply:
Updated.